src/getCoinCombination.test.js

```diff
@@ -7,4 +7,32 @@ describe('getCoinCombination', () => {
     expect(getCoinCombination)
       .toBeInstanceOf(Function);
   });
+
+  test(`should return an array`, () => {
+    expect(getCoinCombination(1))
+      .toEqual([
+        1, 0, 0, 0,
+      ]);
```

Comment:
it is a checks from example. Add your own

Reply:
done